app/sell/index.js

```diff
@@ -1,100 +1,165 @@
-import React from 'react'
-import { StyleSheet, View, Text, Dimensions, ScrollView } from 'react-native'
-import { Link } from 'expo-router'
+import React, { useEffect } from 'react'
+import {
+    Image,
+    StyleSheet,
+    View,
+    Text,
+    Dimensions,
+    ScrollView,
+    TouchableOpacity
+} from 'react-native'
+import Ionicons from '@expo/vector-icons/Ionicons'
+import { useRouter } from 'expo-router'
+import { useDispatch, useSelector } from 'react-redux'
 
+import Divider from '@components/Divider'
+import ErrorDialog from '@components/ErrorDialog'
 import Page from '@components/Page'
+import PaymentSetupRedirect from '@components/PaymentSetupRedirect'
+import {
+    claimOrder,
+    getOrders,
+    resetClaimOrderError,
+    selectClaimedOrder,
+    selectClaimedOrderLoading,
+    selectClaimedOrderError,
+    selectOrders,
+    selectOrdersLoading,
+    selectOrdersError,
+    selectProfileData
+} from '@store'
+import { formatTimeWithIntl, isWithin15Minutes } from '@utils'
 
 const { width: screenWidth } = Dimensions.get('window')
 
 const Sell = () => {
-    const options = [
-        {
-            label: 'Chick-fil-A',
-            time: '2024-09-14T19:30:00'
-        },
-        {
-            label: 'Panda',
-            time: '2024-09-14T19:30:00'
-        },
-        {
-            label: 'Dunkin',
-            time: '2024-09-14T19:35:00'
-        },
-        {
-            label: 'Canes',
-            time: '2024-09-14T19:40:00'
-        },
-        {
-            label: 'Pres-Deli',
-            time: '2024-09-14T19:40:00'
-        },
-        {
-            label: 'Julias',
-            time: '2024-09-14T20:00:00'
-        },
-        {
-            label: "Wendy's",
-            time: '2024-09-14T20:00:00'
+    const dispatch = useDispatch()
+    const router = useRouter()
+
+    const orders = useSelector(selectOrders)
+    const ordersLoading = useSelector(selectOrdersLoading)
+    const ordersError = useSelector(selectOrdersError)
+
+    const claimedOrder = useSelector(selectClaimedOrder)
+    const claimedOrderLoading = useSelector(selectClaimedOrderLoading)
+    const claimedOrderError = useSelector(selectClaimedOrderError)
+
+    const profileData = useSelector(selectProfileData)
+
+    const pressHandler = (order) => {
+        dispatch(claimOrder(order))
+    }
+
+    const icons = {
```

Comment:
What do you mean? I'm doubtful that we would want to load the images in a separate constants file, nor am I sure if that will work properly. If you're just talking about the restaurant list, we don't get much benefit from creating a constant, since we have to hardcode the value we pass to `require` unfortunately.